packages/inter-protocol/src/auction/params.js

```diff
@@ -0,0 +1,220 @@
+import {
+  CONTRACT_ELECTORATE,
+  makeParamManager,
+  ParamTypes,
+} from '@agoric/governance';
+import { objectMap } from '@agoric/internal';
+import { TimerBrandShape, TimeMath } from '@agoric/time';
+import { M } from '@agoric/store';
+
+/** @typedef {import('@agoric/governance/src/contractGovernance/typedParamManager.js').AsyncSpecTuple} AsyncSpecTuple */
+/** @typedef {import('@agoric/governance/src/contractGovernance/typedParamManager.js').SyncSpecTuple} SyncSpecTuple */
+
+// TODO duplicated with zoe/src/TypeGuards.js
+export const InvitationShape = M.remotable('Invitation');
+
+// The auction will start at AUCTION_START_DELAY seconds after a multiple of
+// START_FREQUENCY, with the price at STARTING_RATE. Every CLOCK_STEP, the price
+// will be reduced by DISCOUNT_STEP, as long as the rate is at or above
+// LOWEST_RATE, or until START_FREQUENCY has elapsed.
+
+// in seconds, how often to start an auction
+export const START_FREQUENCY = 'StartFrequency';
+// in seconds, how often to reduce the price
+export const CLOCK_STEP = 'ClockStep';
+// discount or markup for starting price in basis points. 9999 = 1 bp discount
+export const STARTING_RATE = 'StartingRate';
+// A limit below which the price will not be discounted.
+export const LOWEST_RATE = 'LowestRate';
+// amount to reduce prices each time step in bp, as % of the start price
+export const DISCOUNT_STEP = 'DiscountStep';
+// VaultManagers liquidate vaults at a frequency configured by START_FREQUENCY.
+// Auctions start this long after the hour to give vaults time to finish.
+export const AUCTION_START_DELAY = 'AuctionStartDelay';
+// Basis Points to charge in penalty against vaults that are liquidated.  Notice
+// that if the penalty is less than the LOWEST_RATE discount, vault holders
+// could buy their assets back at an advantageous price.
+export const LIQUIDATION_PENALTY = 'LiquidationPenalty';
+
+// /////// used by VaultDirector /////////////////////
+// time before each auction that the prices are locked.
+export const PRICE_LOCK_PERIOD = 'PriceLockPeriod';
+
+const RelativeTimePattern = { relValue: M.nat(), timerBrand: TimerBrandShape };
```

Comment:
to replace with https://github.com/Agoric/agoric-sdk/blob/a4017fc3940f8228dcf99ba965ea74b9f4f0435d/packages/time/src/typeGuards.js#L12-L15